tests/tests/operation_tests.cpp

```diff
@@ -6637,6 +6709,19 @@ BOOST_AUTO_TEST_CASE( comment_beneficiaries_apply )
       ACTORS( (alice)(bob)(sam)(dave) )
       generate_block();
 
+      db_plugin->debug_update( [=]( database& db )
+      {
+         db.modify( db.get_dynamic_global_properties(), [=]( dynamic_global_property_object& gpo )
+         {
+            // Update to turn off inflation
```

Comment:
Let's merge inflation branch and merge master to this branch. Then fix this test.